packages/editor/src/App.css

```diff
@@ -1,75 +1,85 @@
 :root {
-  --editor-bg: #f5f5f5;
-  --editor-header-bg: #fcfcfc;
-  --editor-shadow: 0px 9px 27px rgba(0, 0, 0, 0.07), 0px 3.75998px 11.28px rgba(0, 0, 0, 0.0503198),
-    0px 2.01027px 6.0308px rgba(0, 0, 0, 0.0417275), 0px 1.12694px 3.38082px rgba(0, 0, 0, 0.035),
-    0px 0.598509px 1.79553px rgba(0, 0, 0, 0.0282725), 0px 0.249053px 0.747159px rgba(0, 0, 0, 0.0196802);
+  /* ------all colors------*/
+  --body: #1b1b1b;
+  --background: #ffffff;
+  --N25: #fafafa;
+  --N50: #f8f8f8;
+  --N75: #f3f3f3;
+  --N100: #ebebeb;
+  --N200: #e7e7e7;
+  --N300: #dfdfdf;
+  --N400: #bfbfbf;
+  --N500: #b0b0b0;
+  --N600: #a3a3a3;
+  --N700: #858585;
+  --N800: #757575;
+  --N900: #4a4a4a;
 
-  --accordion-header-bg: #ffffff;
-  --accordion-bg: #fcfcfc;
+  --error-color: #ee4a52;
+  --warning-color: #ffac5e;
+  --success-color: #47c46b;
```

Comment:
rename this to something more matching?

Reply:
For example? I actually find the names quite fitting, don't you? The "error-color" is only used for error messages, "warning" only for warnings, and "success" only for successful cases. Otherwise, could I simply omit the "-color"?

Reply:
Also true :)